packages/gatsby-theme/src/components/Share/Share.js

```diff
@@ -13,13 +14,39 @@ const Share = ({ children, ...props }) => {
       }}
       {...props}
     >
-      {children}
+      <Heading
+        sx={{
+          color: "secondary",
+          m: 0,
+          mr: "small",
+          ...sx,
```

Comment:
We should move this sx to an upper level, since right now it would only affect the Heading